docs/src/examples/components/List/Content/ListExampleEndMedia.tsx

```diff
@@ -8,14 +8,17 @@ const ListExample = () => (
     <List.Item
       content="Program the sensor to the SAS alarm through the haptic SQL card!"
       endMedia={ellipsis}
+      selection
```

Comment:
ideally this prop should be propagated from `List` to `List.Item`s when is set for `List`

Reply:
When using children api, we are directly rendering them, and not propagating any props.
{childrenExist(children) ? children : this.renderItems()}

@kuzhelov , @levithomason , @alinais , I will open a separate bug for this.

Reply:
https://github.com/stardust-ui/react/issues/412